.gitignore

```diff
@@ -0,0 +1,12 @@
+cmake_install.cmake
+CMakeCache.txt
+CMakeFiles/
+documentation.list
+icons/
```

Comment:
This directory contains sources too

Reply:
Thanks for pointing these out.

I'm not that familiar with `sfdk` yet (stubborn Qt Creator click-to-run guy) so I guess I have messed something up. I'll check the ignored files again with a clean pull.